chat-client-ui-types/src/uiContracts.ts

```diff
@@ -44,6 +45,7 @@ export type UiMessageCommand =
     | typeof CHAT_OPTIONS
     | typeof COPY_TO_CLIPBOARD
     | typeof DISCLAIMER_ACKNOWLEDGED
+    | typeof PAIR_PROGRAMMING_ACKNOWLEDGED
```

Comment:
What will extension do when it gets this event?

Reply:
store that pair programming has been acknowledged so the next time VSCode launches the chat client it won't be shown